src/io/gff.rs

```diff
@@ -222,58 +222,54 @@ impl<W: io::Write> Writer<W> {
 }
 
 /// A GFF record
-#[derive(Debug, Default, Serialize, Deserialize, Clone)]
+///
+#[derive(
+    Debug,
+    Default,
+    Serialize,
+    Deserialize,
+    Clone,
+    PartialEq,
+    Eq,
+    Getters,
+    Setters,
+    MutGetters,
+    Builder,
+)]
 pub struct Record {
+    #[getset(get, set, get_mut)]
+    #[builder(default)]
     seqname: String,
+    #[getset(get, set, get_mut)]
+    #[builder(default)]
     source: String,
+    #[getset(get, set, get_mut)]
+    #[builder(default)]
     feature_type: String,
+    #[getset(get, set, get_mut)]
+    #[builder(default)]
     start: u64,
+    #[getset(get, set, get_mut)]
+    #[builder(default)]
     end: u64,
+    #[getset(set, get_mut)]
+    #[builder(default = "\".\".to_owned()")]
     score: String,
+    #[getset(set, get_mut)]
+    #[builder(default = "\".\".to_owned()")]
     strand: String,
+    #[getset(set, get_mut)]
+    #[builder(default)]
     frame: String,
+    #[getset(get, set, get_mut)]
+    #[builder(default = "MultiMap::<String, String>::new()")]
```

Comment:
I think MultiMap already implements Default, but I'm not sure. If so (in addition with just annotating `builder`  at the struct level as mentioned in the comment above), you can probably omit this.